src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java

```diff
@@ -178,6 +190,16 @@ private static ConfigDef createConfigDef() {
                 ++orderInGroup,
                 ConfigDef.Width.MEDIUM,
                 "enable exactly once semantics.");
+        configDef.define(INSERT_QUORUM,
```

Comment:
The range between 0 to 10 is problematic for big clusters. i think we can set it to auto and this will do the trick